crates/harp/src/object.rs

```diff
@@ -603,6 +604,18 @@ impl From<&Vec<i64>> for RObject {
     }
 }
 
+impl From<&Vec<f64>> for RObject {
+    fn from(values: &Vec<f64>) -> Self {
+        unsafe {
+            let vector = RObject::from(Rf_allocVector(REALSXP, values.len() as isize));
+            for idx in 0..values.len() {
+                SET_REAL_ELT(vector.sexp, idx as isize, values[idx] as c_double);
```

Comment:
But I guess then we'd have to move from `from(values: &Vec<f64>)` to `try_from(values: &Vec<f64>)` so maybe we shouldn't worry about that right now.

(i do think maybe we should move to `try_from` in general, since allocations can always be a point of failure...)